test/run.sh

```diff
@@ -7,10 +7,21 @@ if [[ ${CI} == "true" && "${CIRCLE_JOB}" != "x86_64" ]]; then
 fi
 
 # Install necessary additional components for testing
-apt-get -qq install dnsutils -y > /dev/null
+apt-get -qq install dnsutils libcap2-bin -y > /dev/null
 
-# Create necessary directories
+# Create pihole user
+useradd -m -s /bin/bash pihole
```

Comment:
This will create the user with a login shell and home directory, but the `pihole` user is supposed to be a nologin user.